src/remove-dulplicates/index.js

```diff
@@ -1,3 +1,52 @@
-function removeDuplicates(obj) {}
+function removeDuplicates(obj) 
+{
+    /*variable to get the object keys(i.e properties). The function Object.key() by default 
+    *returns array of all the object keys sorted in ascending order when the keys are numbers 
+    *(i.e 1,2,3,...) or quoted(string) numbers (i.e "1","2","3",...)*/
+    let keys = Object.keys(obj);
+
+    //loop through the object keys referencing the values and perform the underly operation
+    for (let k = 0; k < keys.length; k++)
+    {
+        //within the value of each key that is an array, loop through the elements 
+        for (let i = 0; i < obj[keys[k]].length; i++)
+        {
+            //if any element has more than one index... 
+            if (obj[keys[k]].indexOf(obj[keys[k]][i]) !== i)
```

Comment:
All the nested braces make it hard to follow what's happening. You might want to extract to variables.